src/main/kotlin/org/jetbrains/exposed/sql/Table.kt

```diff
@@ -402,6 +403,24 @@ open class Table(name: String = ""): ColumnSet(), DdlAware {
         index(true, *columns)
     }
 
+    /**
+     * Creates a check constraint in this column
+     * @param name The name to identify the constraint, optional.
+     * @param op The expression that the value of this column must satisfy
+     */
+    fun <T> Column<T>.check(name: String = "", op: SqlExpressionBuilder.(Column<T>) -> Op<Boolean>) = apply {
+        table.checkConstraints.add(name to SqlExpressionBuilder.op(this))
```

Comment:
Maybe it's worth to check that constraint with the same name already in `checkConstraints` list?

Reply:
Yes, you are right. I just checked and SQLite is the only database, among MySQL, H2 and Postgres, that supports multiple constraints with the same name.

I'll add the necessary code to check for repeated names.

Reply:
I am also considering to `.trim()` the name before it is added to the list. What do you think @Tapac?

Also, empty names will be added even if there are constraints with empty names already, so an automatic name is generated by the database.